### PR TITLE
Add maximums on how much data we will read for OCSP requests

### PR DESCRIPTION
**Pull request by stevendpclark**
Limit the maximum number of bytes we will consume from the post body, the get request I believe is already a bit too late, but checks added to be consistent. 

Not really sure if this is deserving of a test beyond what we already have.

### CODE REVIEW

File: builtin/logical/pki/ocsp.go [hunk 1]
```diff
@@ -26,6 +27,7 @@ import (
 const (
 	ocspReqParam            = "req"
 	ocspResponseContentType = "application/ocsp-response"
+	maximumRequestSize      = 87 * 5 // A normal simple request is 87 bytes, so give us some buffer
```
Comment (cipherboy) [inline builtin/logical/pki/ocsp.go:30]: We could probably safely bump this to say, 2048 just to ensure we give ourselves more space if necessary for spec changes (or if we start accepting more req extensions)

**Review by kitography — APPROVED** [review]
LGTM!

File: builtin/logical/pki/ocsp.go (hunk 1 above)
Comment (kitography) [reply builtin/logical/pki/ocsp.go:30]: I don't disagree, though I think @stevendpclark might give us a way to stop people from trying to check a bunch of serial numbers at once, and this won't break anything for a new feature.

Comment (stevendpclark) [reply builtin/logical/pki/ocsp.go]: Sure, we could lower/up it as we go, basically just a safety valve, bumped to 2048 for now.

**Review by cipherboy — APPROVED** [review]

**Pull request merged**